examples/07_nvs_read_write/main/nvs_read_write.c

```diff
@@ -0,0 +1,62 @@
+/* Non-Volatile Storage (NVS) Read and Write Example
+
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+*/
+#include <stdio.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "esp_system.h"
+#include "nvs_flash.h"
+#include "nvs.h"
+
+void app_main()
+{
+    nvs_flash_init();
+    system_init();
```

Comment:
`system_init` not needed here since WiFi stack is not used. Plus, this function will be removed shortly.